src/d3a/models/market/balancing.py

```diff
@@ -42,18 +42,22 @@ def __init__(self, time_slot=None, area=None, notification_listener=None, readon
 
         super().__init__(time_slot, area, notification_listener, readonly)
 
-    def offer(self, price: float, energy: float, seller: str):
+    def offer(self, price: float, energy: float, seller: str, iaaFee: bool = False):
```

Comment:
Please use snake_case for variables and argument names. I wonder why flake did not complain about that though.